db/migrate/20181026173627_create_rents.rb

```diff
@@ -0,0 +1,19 @@
+class CreateRents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rents do |t|
+      t.date :start_date, :null => false, :default => Time.now
```

Comment:
reemember to use the new convention for the assignation, using `:` instead of `=>`  for example, `null: false`